apps/ergw_core/src/gtp_context.erl

```diff
@@ -324,12 +324,13 @@ port_message(Server, Request, Msg, Resent) ->
 
 callback_mode() -> [handle_event_function, state_enter].
 
-init([Socket, Info, Version, Interface,
-      #{node_selection := NodeSelect,
-	aaa := AAAOpts} = Opts]) ->
+init({[Socket, Info, Version, Interface,
```

Comment:
Maybe rename "Interface" to "InterfaceM" and add some doc, just to make sure the casual reader realize that this holds a module ref that determines the behaviour of the gtp_context.